### ISSUE TITLE
v0.2b 6D FOF hard-coded

### ISSUE THREAD

**Issue by kusmic** [dnarayanan/caesar#68]
Hello,
While looking around, I noticed that in caesar/caesar/fof6d.py lines 143 and 144 that the linking lengths are hard-coded for position to be b_galaxy * MIS and linking length for velocity is 1 code_velocity_units. I was confused if this is intentional, since the fof6d_LL_ arguments can be defined by the user.

**Comment by kusmic** [comment]
Sorry for the confusion, I didn't know if providing a value for the fof6d_LL_ will change the linking length. Can you confirm if it does and how?

**Comment by weiguangcui** [comment]
No, it won't, but you can add these linking lengths to the input parameters of `run_fof6d`, and then you can give them whatever values you want.
